sdk/core/core-http/lib/atomXmlSerializer.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { HttpOperationResponse } from "./httpOperationResponse";
+
+export interface AtomXmlSerializer {
+  serialize(requestBodyInJson: any): string;
+
+  deserialize(response: HttpOperationResponse): Promise<HttpOperationResponse>;
```

Comment:
Since atom xml operations will be a small percentage of the use cases of the core-http library, I feel a little uncomfortable littering multiple files around core-http for it if we can help it. Can you consider moving this interface definition inside the `atomXmlOperationSpec.ts` file since the serializer is used always from inside the operation spec and because this is only an interface and not the implementation?

Reply:
updated